src/dcap_provider.cpp

```diff
@@ -1157,20 +1485,6 @@ static std::string build_enclave_id_url(
     return qe_id_url.str();
 }
 
-static std::unique_ptr<std::vector<uint8_t>> try_cache_get(

```

Comment:
This function has been moved higher in the file so that the region url function can use it